tests/test_decrypter.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+from sdc.crypto.decrypter import decrypt
+from sdc.crypto.key_store import KeyStore
+from sdc.crypto.exceptions import InvalidTokenException
+from tests import TEST_DO_NOT_USE_UPSTREAM_PUBLIC_PEM, TEST_DO_NOT_USE_SR_PRIVATE_PEM
+from tests import TOO_FEW_TOKENS_JWE, VALID_JWE
+
+
+KEY_PURPOSE_AUTHENTICATION = "authentication"
+
+# jwt.io public key signed
+TEST_DO_NOT_USE_PUBLIC_KEY = """-----BEGIN PUBLIC KEY-----
+MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDdlatRjRjogo3Wojg
+GHFHYLugdUWAY9iR3fy4arWNA1KoS8kVw33cJibXr8bvwUAUparCwlv
+dbH6dvEOfou0/gCFQsHUfQrSDv+MuSUMAe8jzKE4qW+jK+xQU9a03GU
+nKHkkle+Q0pX/g6jXZ7r1/xAK5Do2kQ+X5xK9cipRgEKwIDAQAB
+-----END PUBLIC KEY-----"""
+
+
+class TestDecrypter(TestCase):
```

Comment:
Since we are using pytest now, you could remove all of this inheritance and assertEqual stuff.

Reply:
Done